### DISCUSSION TITLE
JPCMG isn't working

### DISCUSSION THREAD

**Discussion by Hilololsusy** [3kh0/ext-remover#629] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I have Connect for Chrome - Education
Ext id:  ddfbkhpmcdbciejenfcolaaiebnjcbfc
Webstore link:
https://chrome.google.com/webstore/detail/connect-for-chrome-educat/ddfbkhpmcdbciejenfcolaaiebnjcbfc
ChromeOS version: 112.0.5615.134

That was probably not needed but tell you about my things.
When I go to chrome://serviceworker-internals/, I can not find my extension. Anyone have a fix?

**Comment by S-PScripts** [comment]
we do not. use another exploit


(necropost to mark an answer)
